File: src/main/java/noteedit/NoteEditApplication.java
package noteedit;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
@MapperScan("noteedit.mapper")
public class NoteEditApplication {

    public static void main(String[] args) {
        SpringApplication.run(NoteEditApplication.class, args);
    }

}

File: src/main/java/noteedit/NoteEditApplication.kt
package noteedit

import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
@MapperScan("noteedit.mapper")
open class NoteEditApplication

fun main(args: Array<String>) {
    runApplication<NoteEditApplication>(*args)
}